docs/source/conf.py

```diff
@@ -195,6 +196,24 @@
     "torch": ("https://pytorch.org/docs/stable/", None),
 }
 
+# Redirects for moved pages
+redirects = {
+    "getting-started-setup": "getting-started.html",
+    "export-overview": "using-executorch-export.html",
+    "runtime-build-and-cross-compilation": "using-executorch-building-from-source.html",
+    "tutorials/export-to-executorch-tutorial": "../using-executorch-export.html",
+    "running-a-model-cpp-tutorial": "using-executorch-cpp.html",
+    "build-run-vulkan": "backends-vulkan.html",
+    "executorch-arm-delegate-tutorial": "backends-arm-ethos-u.html",
+    "build-run-coreml": "backends-coreml.html",
+    "build-run-mediatek-backend": "backends-mediatek.html",
+    "build-run-mps": "backends-mps.html",
+    "build-run-qualcomm-ai-engine-direct-backend": "backends-qualcomm.html",
+    "build-run-xtensa": "backends-cadence.html",
+    "apple-runtime": "using-executorch-ios.html",
+    "tutorials/devtools-integration-tutorial": "../using-executorch-troubleshooting.html",
```

Comment:
I don't think this is correct. original path is correct, it will not show up until running `tutorial_source/devtools-integration-tutorial.py`. Let me fix that in my PR.

Reply:
yes your redirection is working, but the page you redirect to is not the one we need.

this is the one you redirected to: https://pytorch.org/executorch/0.6/using-executorch-troubleshooting.html
but this is the one we need: https://pytorch.org/executorch/0.6/tutorials/devtools-integration-tutorial.html

Reply:
Sure. Feel free to update the redirect link if you prefer that page.